built/History.js

```diff
@@ -4,22 +4,22 @@ var __extends = (this && this.__extends) || function (d, b) {
     function __() { this.constructor = d; }
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
-var React = require('react');
```

Comment:
I am in 2.0.9 for tsc.exe if that helps tracking it down. I am not in the default 2.0.1 of vscode and did not go in 2.1 yet due to current project in prod where I need to triple check for breaking changes around constrctor initilization and the extents of built-in types.